docs/tutorials/abn/abn.md

```diff
@@ -2,173 +2,185 @@
+Once the candidate version is ready, `Lookup()` will return both indices (0 and 1) so that requests can be distributed across versions.
+
+## Compare versions using Grafana
 
-## Launch experiment
+Inspect the metrics using Grafana. If Grafana is deployed to your cluster, port-forward requests as follows:
 
 ```shell
-iter8 k launch \
---set abnmetrics.application=default/backend \
---set "tasks={abnmetrics}" \
---set runner=cronjob \
---set cronjobSchedule="*/1 * * * *"
+kubectl port-forward service/grafana 3000:3000
 ```
 
-??? note "About this experiment"
-    This experiment periodically (in this case, once a minute) reads the `abn` metrics associated with the *backend* application component in the *default* namespace. These metrics are written by the frontend component using the *WriteMetric()* interface as a part of processing user requests.
-
-## Inspect experiment report
-
-Inspect the metrics:
+Open Grafana in a browser:
 
 ```shell
-iter8 k report
+http://localhost:3000/
 ```
 
-??? note "Sample output from report"
-    ```
-    Experiment summary:
-    *******************
-
-    Experiment completed: true
-    No task failures: true
-    Total number of tasks: 1
-    Number of completed tasks: 1
-    Number of completed loops: 3
-
-    Latest observed values for metrics:
-    ***********************************
-
-    Metric                   | backend (v1) | backend-candidate-1 (v2)
-    -------                  | -----        | -----
-    abn/sample_metric/count  | 35.00        | 28.00
-    abn/sample_metric/max    | 99.00        | 100.00
-    abn/sample_metric/mean   | 56.31        | 52.79
-    abn/sample_metric/min    | 0.00         | 1.00
-    abn/sample_metric/stddev | 28.52        | 31.91
-    ```
-The output allows you to compare the versions against each other and select a winner. Since the experiment runs periodically, the values in the report will change over time.
+[Add a JSON API data source](http://localhost:3000/connections/datasources/marcusolsson-json-datasource) `Iter8` with:
 
-Once a winner is identified, the experiment can be terminated, the winner can be promoted, and the candidate version(s) can be deleted.
+- URL `http://iter8.default:8080/metrics` and 
+- query string `application=default%2Fbackend`
 
-To delete the experiment:
+[Create a new dashboard](http://localhost:3000/dashboards) by *import*. Do so by pasting the contents of this [JSON definition](https://gist.githubusercontent.com/Alan-Cha/aa4ba259cc4631aafe9b43500502c60f/raw/034249f24e2c524ee4e326e860c06149ae7b2677/gistfile1.txt) into the box and *load* it. Associate it with the JSON API data source defined above.
 
-```shell
-iter8 k delete
-```
+The Iter8 dashboard allows you to compare the behavior of the two versions of the backend component against each other and select a winner. Since user requests are being by the load generation script, the values in the report may change over time.
```

Comment:
```suggestion
The Iter8 dashboard allows you to compare the behavior of the two versions of the backend component against each other and select a winner. Since user requests are being sent by the load generation script, the values in the report may change over time.
```